### DISCUSSION TITLE
IntelliJ IDEA LigthEdit mode support

### DISCUSSION THREAD

**Discussion by linux-china** [community/community#9262] [Copilot Conversations] [answered]
Github Copilot is available with [IntelliJ IDEA LightEdit mode﻿](https://www.jetbrains.com/help/idea/lightedit-mode.html) ,   and it's good to edit single File and without worry about GitHub Copilot telemetry problem.   But code completion can not work well for some cases. Screenshot as following: 

![Snip20211223_141](https://user-images.githubusercontent.com/46711/147296783-4f6a5841-cfe9-4dbc-b7ca-9bf93b535255.png)

After tab pressed: 

![Snip20211223_142](https://user-images.githubusercontent.com/46711/147296795-c10946bd-7ca6-48b2-a45c-7874eade5421.png)

Maybe more testing for LightEdit mode :)   Thanks.

**Comment by jansorg** [comment, marked as answer]
@linux-china Could you add why light edit mode is different in regards to telemetry? 
We'll take a closer look at integration with LightEdit mode, thanks!

**Reply by linux-china** [reply]
I summited the issue to JetBrains, and this is the issue link https://youtrack.jetbrains.com/issue/IDEA-285728

**Reply by jansorg** [reply]
Thanks, JetBrains is right that this is a bug in the GitHub Copilot plugin. We'll fix it for one of the next updates.
